RecoJets/JetProducers/python/hltMVAJetPuId_cfi.py

```diff
@@ -0,0 +1,28 @@
+from RecoJets.JetProducers.hltPUIdAlgo_cff import *
+
+MVAJetPuIdCalculator = cms.EDProducer('MVAJetPuId',
```

Comment:
the renaming of the file should have come with renaming of the modules as well:
the cfi file should have a definition only of one module instance and this module should have the same name as the file itself.